FrejaEidClient/src/main/java/com/verisec/frejaeid/client/client/impl/BasicClient.java

```diff
@@ -86,13 +90,19 @@ public GenericBuilder(String keystorePath, String keystorePass, String certifica
             if (numberOfFails == KeyStoreType.values().length) {
                 throw new FrejaEidClientInternalException(String.format("Failed to initiate SSL context with supported keystore types %s.", KeyStoreType.getAllKeyStoreTypes()));
             }
+            setServerCustomUrl(frejaEnvironment);
+        }
+
+        private void setServerCustomUrl(FrejaEnvironment frejaEnvironment) {
+            LOG.debug("Setting {} freja environment {}", frejaEnvironment);
```

Comment:
Remove one pair of braces or add missing parameter.

Reply:
Maybe Freja instead of freja?

Reply:
Done.